plugin/js/import.js

```diff
@@ -68,20 +69,21 @@ function processNextItem(firstOrLast, buildAjaxCallback, callerButton, url_base)
 
     let resultElement = elementToProcess.closest('.list-group-item-action').find('.process-result');
     let itemType      = elementToProcess.data('itemtype');
-    let ajaxUrl       = url_base + '/ajax/' + buildAjaxCallback(itemType);
+    let ajaxUrl       = url_base + '/ajax/' + buildAjaxCallback(itemType, startFromId);
     resultElement.addClass('fa-spinner fa-pulse');
     resultElement.attr('title', '...');
 
     $.get(
         ajaxUrl,
         function (data) {
             resultElement.removeClass('fa-spinner fa-pulse');
-            resultElement.closest('.list-group-item-action').find('.form-check-input').removeClass('to-process');
 
             if (data === 'OK') {
+                resultElement.closest('.list-group-item-action').find('.form-check-input').removeClass('to-process');
+                resultElement.removeClass('fa-circle-xmark fa-regular text-danger ' + itemType);
                 resultElement.addClass('fa-circle-check fa-regular text-success ' + itemType);
                 resultElement.attr('title', '');
-                processNextItem(firstOrLast, buildAjaxCallback, callerButton, url_base);
+                processNextItem(firstOrLast, buildAjaxCallback, callerButton, url_base, startFromId);
```

Comment:
When we precess the next item, we always have to start from 0, don't we?
 ```suggestion
                processNextItem(firstOrLast, buildAjaxCallback, callerButton, url_base, 0);
```

Reply:
`startFromId` is 0 by default and later we might want to start from the last imported item, not always from 0.

Reply:
This is the case when OK was returned, so from this point we must start a new import, from 0.